industry_benchmarks/utils/extras/extract_results.py

```diff
@@ -108,6 +115,27 @@ def extract(results_0, results_1, results_2, output):
                   f'Missing results have been found for {missing_files}.')
         raise ValueError(errmsg)
 
+    # Now that we know all the files exist that we expect, lets check for errors
+    click.echo("Checking files for errors...")
+    has_errors = False
+    for file in tqdm(files_0 + files_1 + files_2):
+        with open(file, 'r') as fd:
+            result = json.load(fd, cls=JSON_HANDLER.decoder)
+        if result['estimate'] is None or result['uncertainty'] is None:
+            has_errors = True
+            problem_files.append(f)
```

Comment:
```suggestion
```
This variable doesn't exist and seems to be doing nothing?

Reply:
Should be good now!